src/covariance.jl

```diff
@@ -16,11 +16,52 @@ Evaluate the covariance at objects `x₁` and `x₁`.
 """
 (cov::Covariance)(x₁, x₂) = sill(cov.γ) - cov.γ(x₁, x₂)
 
+"""
+    isstationary(cov)
+
+Check if covariance `cov` possesses the 2nd-order stationary property.
+"""
+isstationary(cov::Covariance) = isstationary(typeof(cov.γ))
+
+"""
+    isisotropic(cov)
+
+Tells whether or not covariance `cov` is isotropic.
+"""
+isisotropic(cov::Covariance) = isisotropic(cov.γ.ball)
```

Comment:
We cannot use an internal field of gamma here. Please try to only use the public api of gamma in the functions of cov.